app/views/layout.jade

```diff
@@ -51,8 +52,16 @@ html(itemscope, itemtype='http://schema.org/Product')
 			window.csrfToken = "#{csrfToken}";
 			
 		block scripts
-		script(src=buildJsPath("libs/jquery-1.11.1.min.js", {fingerprint:false}))
 		script(src=buildJsPath("libs/angular-1.3.15.min.js", {fingerprint:false}))
+		script(src=buildJsPath("libs/jquery-1.11.1.min.js", {fingerprint:false}))
```

Comment:
Any reason to load jquery after angular, or is it just an accidental move? I'd move it back if not, just in case it matters.